File: client/core/api/models/template.ts
import {
  FormBuilder,
  FormGroup,
  Validators
} from '@angular/forms';

import { Organization } from './organization';
import { TemplatePlane } from './template-plane';

export interface Template {
  id: number;
  organizationId: number;
  title: string;
  description: string;

  organization: Organization;

  templatePlanes: TemplatePlane[];
}

export const TemplateForm = (template: Template, fb: FormBuilder): FormGroup =>
  fb.group({
    id: [template.id],
    organizationId: [template.organizationId, Validators.required],
    title: [template.title, Validators.required],
    description: [template.description]
  })

File: server/AppSupport.Data/Entities/Template.cs
using System;
using System.Collections.Generic;

namespace AppSupport.Data.Entities
{
    public class Template
    {
        public int Id { get; set; }
        public int OrganizationId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }

        public Organization Organization { get; set; }

        public IEnumerable<TemplatePlane> TemplatePlanes { get; set; }
    }
}